controls/scheduler/customdateformats.md

```diff
@@ -0,0 +1,44 @@
+---
```

Comment:
The page doesn't have an inline ToC ("In this article", in the sidebar). That's due to the heading levels. Consider switching your H3s to H2s.

Reply:
Agree